cpp/CMakeLists.txt

```diff
@@ -480,6 +483,7 @@ target_compile_definitions(cudf PRIVATE "JITIFY_PRINT_LOG=0")
 target_include_directories(cudf
            PUBLIC      "$<BUILD_INTERFACE:${DLPACK_INCLUDE_DIR}>"
                        "$<BUILD_INTERFACE:${JITIFY_INCLUDE_DIR}>"
+                       "$<BUILD_INTERFACE:${CUCO_INCLUDE_DIR}>"
```

Comment:
You should add `cuco::cuco` to the target_link_libraries call of `cudf` instead of this approach. If cuco is found locally on the machine the variable `cuco_SOURCE_DIR` won't be set and you will have an invalid path for `CUCO_INCLUDE_DIR`